### PR TITLE
feat: Add support gruf 2.19

### PR DESCRIPTION
**Pull request by AS-AlStar**
Update appraise for gruf 2.19 and old versions

### CODE REVIEW

File: instrumentation/gruf/lib/opentelemetry/instrumentation/gruf/version.rb [hunk 1]
```diff
@@ -7,7 +7,7 @@
 module OpenTelemetry
   module Instrumentation
     module Gruf
-      VERSION = '0.1.1'
+      VERSION = '0.1.2'
```
Comment (arielvalentin) [inline instrumentation/gruf/lib/opentelemetry/instrumentation/gruf/version.rb:10]:
```suggestion
      VERSION = '0.1.1'
```
Our release process manages gem versions. No need to change this manually

File: instrumentation/gruf/Appraisals [hunk 2]
```diff
@@ -4,10 +4,7 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
-appraise 'gruf-2.15.1' do
```
Comment (arielvalentin) [inline instrumentation/gruf/Appraisals:7]: This means we lose test coverage for versions prior to 2.19.

Is that ok? Are you looking to drop support for older versions?

**Review by arielvalentin — CHANGES_REQUESTED** [review]
This means we won't have test coverage for versions earlier than 2.19

File: instrumentation/gruf/lib/opentelemetry/instrumentation/gruf/version.rb (hunk 1 above)
Comment (AS-AlStar) [reply instrumentation/gruf/lib/opentelemetry/instrumentation/gruf/version.rb]: ok

File: instrumentation/gruf/Appraisals (hunk 2 above)
Comment (AS-AlStar) [reply instrumentation/gruf/Appraisals:7]: I added the support of the last three versions of Gruf, it seems that there is no need to indicate the dependence of the GRPC version

Comment (arielvalentin) [reply instrumentation/gruf/Appraisals:7]: thank you!

**Comment by arielvalentin** [comment]
Please remove the following to re-enable the tests in CI: 

https://github.com/open-telemetry/opentelemetry-ruby-contrib/blob/a005a987c74f1dd0958aae9e4d47ad84cf34a611/.github/workflows/ci-instrumentation.yml#L55
https://github.com/open-telemetry/opentelemetry-ruby-contrib/blob/a005a987c74f1dd0958aae9e4d47ad84cf34a611/.github/workflows/ci-instrumentation-canary.yml#L64

**Review by arielvalentin — APPROVED** [review]

**Pull request merged**